tests/compas/datastructures/test_mesh.py

```diff
@@ -509,7 +509,7 @@ def test_face_coordinates():
 
 def test_face_normal():
     mesh = Mesh.from_obj(compas.get('quadmesh.obj'))
-    assert mesh.face_normal(0) == [0.5435358481001584, -0.16248515023849733, 0.8235091728584537]
+    assert mesh.face_normal(0) == [0.54353584810015843, -0.1624851502384973, 0.8235091728584536]
```

Comment:
this should be
```python
from compas.geometry import allclose
assert allclose(mesh.face_normal(0), [0.543535, -0.162485, 0.823509])
```

Reply:
with the import somewhere at the top of the file of course.

Reply:
we should also stop using these hardcoded identifiers that assume some kind of specific ordering of the internal data. better to use a mesh of which we actually know the normals, like a cube...